const displayError = (a, b, data) => {
    console.log("checkStatus arrow function called");
    error.textContent = '';
    return x ** y;
    alert('Action completed!');
};